### DISCUSSION TITLE
Change the output directory for oasis rmp files

### DISCUSSION THREAD

**Discussion by mzapponi** [esm-tools/esm_tools#900] [ESM-Tools support]
I am running an initialisation run with AWI-CM3 on levante, 

during which oasis produces the remapping files rmp* in the experiment_id/run_*_*/work folder to then copy them in pool directory (that now is Jan Streffing directory) , but since i don't have permission to write in that folder they won't be copied. 

So my question would be, how can I modify the output pool directory for oasis so that the files are copied in one of my folder?

Once they are checked to be correctly produced they can be copied in the pool directory i think.

**Comment by mandresm** [comment]
In this case you can add the following to your runscript:
```yaml
oasis3mct:
    pool_dir: <absolute_path_to_a_folder_of_your_choice>
```

This should copy the files there. And just a note, I would recommend to use this exclusively for development of oasis restarts/remappings, but this shouls end up in the pool once things are working. @JanStreffing, any comments?
